app/controllers/weather_controller.rb

```diff
@@ -10,6 +10,7 @@ def index
 			location = parsed_json['location']['city']
 			temp_f = parsed_json['current_observation']['temp_f']
```

Comment:
here in the index method of the weather_controller you're creating the cariable temp_f which holds the temperature for Austin. They way you pass variables to the view is by making temp_f on line 11 an instance variables(@temp_f)
```@temp_f = parsed_json['current_observation']['temp_f']```